### ISSUE TITLE
hip_runtime.h' file not found

### ISSUE THREAD

**Issue by DocMAX** [pappacena/pytorch-rocm-gtt#2]
```
(.venv) docmax@ai:~/logai$ python logai-gpu.py
/home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/gttalloc.c:5:10: fatal error: 'hip/hip_runtime.h' file not found
    5 | #include <hip/hip_runtime.h>
      |          ^~~~~~~~~~~~~~~~~~~
1 error generated.
failed to execute:/opt/rocm/lib/llvm/bin/clang++  -O3 --driver-mode=g++ -O3 -Llib --hip-link  -x c /home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/gttalloc.c -o "/home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/gttalloc.so" -shared -fPIC
Traceback (most recent call last):
  File "/home/docmax/logai/logai-gpu.py", line 2, in <module>
    pytorch_rocm_gtt.patch()
  File "/home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/__init__.py", line 12, in patch
    compile()
  File "/home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/__init__.py", line 26, in compile
    subprocess.run(
  File "/usr/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['hipcc', '/home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/gttalloc.c', '-o', '/home/docmax/logai/.venv/lib/python3.10/site-packages/pytorch_rocm_gtt/gttalloc.so', '-shared', '-fPIC']' returned non-zero exit status 1.
```
ROCm is in /opt/rocm(/include...). Any idea how to make it find hip_runtime.h?

**Comment by DocMAX** [comment]
Ok, fixed it with option -I/opt/rocm/include. But now i get:
```
(.venv) docmax@ai:~/logai$ ROCM_PATH=/opt/rocm python logai-gpu.py
Using GPU for inference.
AI Question-Answer System (GPT-Neo with ROCm)

Enter your question (or type 'exit' to quit): Who is Hitler?
Traceback (most recent call last):
  File "/home/docmax/logai/logai-gpu.py", line 69, in <module>
    main()
  File "/home/docmax/logai/logai-gpu.py", line 65, in main
    answer = generate_answer(model, tokenizer, question)
  File "/home/docmax/logai/logai-gpu.py", line 26, in generate_answer
    outputs = model.generate(
  File "/home/docmax/logai/.venv/lib/python3.10/site-packages/torch/utils/_contextlib.py", line 116, in decorate_context
    return func(*args, **kwargs)
  File "/home/docmax/logai/.venv/lib/python3.10/site-packages/transformers/generation/utils.py", line 1829, in generate
    self._prepare_special_tokens(generation_config, kwargs_has_attention_mask, device=device)
  File "/home/docmax/logai/.venv/lib/python3.10/site-packages/transformers/generation/utils.py", line 1678, in _prepare_special_tokens
    and isin_mps_friendly(elements=eos_token_tensor, test_elements=pad_token_tensor).any()
  File "/home/docmax/logai/.venv/lib/python3.10/site-packages/transformers/pytorch_utils.py", line 328, in isin_mps_friendly
    return torch.isin(elements, test_elements)
RuntimeError: HIP error: invalid device function
HIP kernel errors might be asynchronously reported at some other API call, so the stacktrace below might be incorrect.
For debugging consider passing AMD_SERIALIZE_KERNEL=3
Compile with `TORCH_USE_HIP_DSA` to enable device-side assertions.
```

**Comment by DocMAX** [comment]
Ok fixed with HSA_OVERRIDE_GFX_VERSION=9.0.0
